nncf/experimental/torch/fx/nncf_graph_builder.py

```diff
@@ -64,6 +65,22 @@ def _get_layer_attributes(
             )
         return None
 
+    def _map_fx_unique_metatypes(node: torch.fx.Node, metatype: om.OperatorMetatype) -> om.OperatorMetatype:
+        """
+        Attempts to retrieve correct subtype for the given node.
+
+        :param node: Given node.
+        :param metatype: Given node metatype.
+        :param model: Target GraphModule instance.
+        :return: Correct FX metatype of the given node if it is exist or the original node metatype otherwise.
+        """
+        if metatype in [om.PTEmbeddingMetatype]:
+            weight_node = node.args[0]
+            if weight_node.op == "get_attr":
+                return om.FXEmbeddingMetatype
+
```

Comment:
Please add a synthetic model with an embedding to the test_models.py

Reply:
> Alright, I will add a graph test for it? I can use the ShortTransformer() and MultiBranchesConnectedModel() and generate reference graphs.

Same as above